engines/python/src/main/java/ai/djl/python/engine/PyEnv.java

```diff
@@ -157,6 +157,51 @@ public Map<String, String> getInitParameters() {
         return initParameters;
     }
 
+    /**
+     * Create python virtual environment if needed.
+     *
+     * @param name the virtual environment name
+     */
+    public synchronized void createVirtualEnv(String name) {
+        Path path = Utils.getVenvDir().resolve(name).toAbsolutePath();
```

Comment:
Utils.getVenvDir() doesn't seems belong to DJL, we should keep it in Python engine

Reply:
I forgot to create PR to DJL. I added it to api

Reply:
Changed.